kartothek/core/common_metadata.py

```diff
@@ -730,7 +731,12 @@ def empty_dataframe_from_schema(schema, columns=None, date_as_object=False):
     """
 
     if ARROW_LARGER_EQ_0130:
-        df = schema.internal().empty_table().to_pandas(date_as_object=date_as_object)
+        df = (
+            schema.remove_metadata()
```

Comment:
Can you remove this again? I think I'd prefer using the `_ensure_string_type` mapping for both code paths. This reduces the complexity since both paths use the same logic